services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/liitteet/HankeAttachment.kt

```diff
@@ -0,0 +1,13 @@
+package fi.hel.haitaton.hanke.liitteet
+
+import java.time.LocalDateTime
+import java.util.*
+
+data class HankeAttachment(
+    val id: UUID?,
+    val name: String?,
+    val user: String?,
+    var created: LocalDateTime?,
+    var tila: AttachmentTila?,
+    var hankeId: Int?,
```

Comment:
Why are these three vars?

Reply:
Changed hankeId to hankeTunnus. Changed the names too.

Reply:
Yes, but why aren't they vals like the first three?